.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+__tasks:__
+
+- [ ] test coverage
+
+__information:__
+
+| q                | a
+| ---------------- | ---
+| Fixed tickets    |
```

Comment:
I'd remove the spaces before the `|`. GitHub's editor uses a non-monospaced font, in which the seemingly random spaces look a bit weird.